File: VB/XtraReport1.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraReports.UI
' ...

Namespace MailMergeSummary
	Partial Public Class XtraReport1
		Inherits DevExpress.XtraReports.UI.XtraReport
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub xrLabel2_SummaryCalculated(ByVal sender As Object, ByVal e As TextFormatEventArgs) Handles xrLabel2.SummaryCalculated
			e.Text = String.Format("Category #{0} contains {1} items.", (CType(sender, XRLabel)).Tag, e.Value)
		End Sub

	End Class
End Namespace

File: CS/XtraReport1.cs
using System;
using DevExpress.XtraReports.UI;
// ...

namespace MailMergeSummary {
    public partial class XtraReport1 : DevExpress.XtraReports.UI.XtraReport {
        public XtraReport1() {
            InitializeComponent();
        }

        private void xrLabel2_SummaryCalculated(object sender, TextFormatEventArgs e) {
            e.Text = String.Format("Category #{0} contains {1} items.", ((XRLabel)sender).Tag, e.Value);
        }

    }
}
